doc/sources/index.rst

```diff
@@ -49,38 +49,79 @@ Important Links
 - `About daal4py <https://github.com/intel/scikit-learn-intelex/tree/master/daal4py>`_
 
 .. toctree::
-   :caption: About
+   :caption: Getting Started
```

Comment:
we probably should discuss on discussing this menu separately - not 100% like this grouping overall.

Reply:
Sure Nikolay, We tried to match the topics as close as the ipex github.io documentation, to maintain consistency. We can publish this as the first version for now and start putting together a reconciled format for the next release of the documentation

Reply:
Alternatively we can keep current menu format and do incremental versions updates to it.  There are several things like  getting started that should be removed for sure

Reply:
I'm okay with renaming it, just use "Get Started" instead :)